cli.py

```diff
@@ -234,19 +234,19 @@ def get_gravatar_url(email, size):
                         u.info['avatar'] = filename
                         u.info['container'] = container
                         db.session.commit()
-                        print "Done!"
+                        print("Done!")
                     else:
-                        print "No Gravatar, this user will use the placeholder."
+                        print("No Gravatar, this user will use the placeholder.")
                 except:
```

Comment:
- To catch any exception use the ``except Exception``, instead of 
  ``except BaseException`` or just ``except``. 
- Indicate the most specific exception type in ``except`` block. 

```python


     # Bad
     try:
         mapping[key]
     except Exception:
         ...

     # Better
     try:
         mapping[key]
     except KeyError:
         ...
```